src/components/Page/PageBanner.vue

```diff
@@ -1,159 +1,121 @@
 <template>
-    <div class="bannerDiv" v-if="!!banner"  @mouseover="mouseEnter" @mousemove="positionTooltip" @mouseleave="mouseLeave" >
-        <Media :url="banner.url" :title="banner.title" class="bannerImage" />
-        <img v-if="imgTooltip" :src="$resolveURL(imgTooltip)" class="imgTooltip" />
-        <!-- TODO if imgTooltip is always an asset using asset:// would be cheaper -->
-    </div>
+  <div class="bannerDiv" v-if="!!banner"  @mouseover="mouseEnter" @mousemove="positionTooltip" @mouseleave="mouseLeave" >
+    <Media :url="banner.url" :title="banner.title" class="bannerImage" />
+    <img v-if="imgTooltip" :src="$resolveURL(imgTooltip)" class="imgTooltip" />
+    <!-- TODO if imgTooltip is always an asset using asset:// would be cheaper -->
+  </div>
 </template>
 
-
 <script>
 import Media from '@/components/UIElements/MediaEmbed.vue'
 
 export default {
-    name: 'pageBanner',
-    props: ['tab', 'page'],
-    components: {
-        Media
-    },
-    data() {
-        return{     
-            tooltipActive: false,
+  name: 'pageBanner',
+  props: ['tab', 'page'],
+  components: {
+    Media
+  },
+  data() {
+    return {   
+      tooltipActive: false
+    }
+  },
+  computed: {
+    banner() {
+      let num = parseInt(this.page.pageId)
+      if (this.page.theme == 'scratch') {
+        return {
+          url: `/storyfiles/hs2/scratch/${this.page.scratchBanner}`, 
+          title: this.page.scratchTooltip ? this.page.scratchTooltip : undefined
         }
+      }
+      else if (this.page.theme == 'cascade')
+        return {url: '/images/header_cascade.gif', title: undefined}
+      else if (this.page.theme == 'trickster')
+        return {url: '/images/trickster_sitegraphics/menu.swf', title: undefined}
+      else if (this.page.flag.includes('X2COMBO'))
+        return {url: '/images/act6act5act1x2combo.gif', title: undefined}
+      else if (num == 9987)
+        return {url: '/images/collide_header.gif', title: undefined}
+      else if (num == 10027)
```

Comment:
We could probably switch these out with flags instead of referencing actual page numbers